frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/ScopeListContainer/SaveStatus/SaveStatus.test.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { SaveStatus, type SaveStatusProps } from './SaveStatus';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+
+const defaultProps: SaveStatusProps = {
+  isPending: true,
+  isSaved: true,
+};
+
+describe('SaveStatus', () => {
+  it('should render pending status with spinner', () => {
+    renderSaveStatus({
+      componentProps: {
+        isSaved: false,
+      },
```

Comment:
The purpose of using defaultMock as default values is to make the tests clearer. Currently, it appears that the spinner is displayed in the initial state when isSaved is false. However, explicitly defining isPending provides more clarity and improves readability.

```suggestion
      componentProps: {
        isPending: true,
        isSaved: false,
      },
```

Reply:
Totally agree. Removed the default props completely to increase readability for the tests